api/src/test/java/org/pmiops/workbench/cdr/dao/CriteriaDaoTest.java

```diff
@@ -0,0 +1,66 @@
+package org.pmiops.workbench.cdr.dao;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.cdr.model.Criteria;
+import org.pmiops.workbench.testconfig.TestCdrJpaConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = {TestCdrJpaConfig.class})
+@ActiveProfiles("test-cdr")
+public class CriteriaDaoTest {
+
+    @Autowired
+    CriteriaDao criteriaDao;
+    private Criteria icd9Criteria1;
+    private Criteria icd9Criteria2;
+    private Criteria demoCriteria1;
+    private Criteria demoCriteria2;
+
+    @Before
+    public void setUp() {
+        icd9Criteria1 = createCriteria("ICD9", 2);
+        icd9Criteria2 = createCriteria("ICD9", 1);
+        demoCriteria1 = createCriteria("DEMO", 2);
+        demoCriteria2 = createCriteria("DEMO", 1);
+        criteriaDao.save(icd9Criteria1);
+        criteriaDao.save(icd9Criteria2);
+        criteriaDao.save(demoCriteria1);
+        criteriaDao.save(demoCriteria2);
+    }
+
+    @Test
+    public void findCriteriaByParentId() throws Exception {
+        final List<Criteria> icd9List = criteriaDao.findCriteriaByTypeLikeAndParentIdOrderBySortOrderAsc(icd9Criteria1.getType(), 0L);
```

Comment:
If you're going to use "type like" in the DAO, make sure you do a partial match test here.

Reply:
Will do